assembl/static2/js/app/components/common/manageErrorAndLoading.jsx

```diff
@@ -29,6 +32,10 @@ const manageErrorAndLoading = (props: Props) => (WrappedComponent: React.Compone
 
   if (loading || (data && data.loading)) {
     if (props.displayLoader) {
+      if (props.customLoader) {
+        return props.customLoaderProps ? <customLoader {...props.customLoaderProps} /> : <customLoader />;
```

Comment:
customLoader is for me undefined here. It should have been `<props.customLoader`
I think your `DumbHeaderPlaceholder` is actually never rendered, so this is equivalent to `displayLoader: false`

Reply:
Ah you actually used `displayLoader: false` with `customLoader` in `Header` component. So this code is actually never reached and is completely broken. I removed the code in https://github.com/assembl/assembl/commit/ea52a8025a38de465202e2e2e6c4b1d3479124ed